docs/first-use/query.txt

```diff
@@ -180,13 +179,24 @@ CrateDB will respond using ``JSON``. You should see something like this:
 
 In this example, we have a single row with a single column: ``3879``.
 
+Third-Party Client Tools
+------------------------
+
+The admin UI and Crash are the two client tools that ship with CrateDB. But
+those are not the only client tools you can use with CrateDB.
+
+Because CrateDB uses the PostgreSQL wire protocol, most PostgreSQL compatible
+client tools should work with CrateDB. Check out the `clients category`_ on the
+blog for highlighted client tools.
```

Comment:
what about all the drivers (at least maintained by crate)? shouldn't we mention them as well?

Reply:
we get to this on the next page

we also mention at the start of this section that we will get to client libraries next :)

Reply:
ah right :)